src/v2/Apps/Order/Routes/__tests__/Offer.jest.tsx

```diff
@@ -286,10 +316,9 @@ describe("Offer InitialMutation", () => {
 
     it("tracks the offer input focus", () => {
       expect(mockPostEvent).not.toHaveBeenCalled()
-
+      page.selectCustomAmount()
       page.find("input").simulate("focus")
 
-      expect(mockPostEvent).toHaveBeenCalledTimes(1)
```

Comment:
We would be now calling 3 tracking events by this point (see tests for `PriceOptions` component).